packages/react-components/src/components/VideoGallery.tsx

```diff
@@ -101,7 +111,7 @@ export interface VideoGalleryProps {
    */
   styles?: VideoGalleryStyles;
   /** Layout of the video tiles. */
-  layout?: 'default' | 'floatingLocalVideo';
+  layout?: LocalTileMode;
```

Comment:
The name of this field and type don't match. This is a smell that one or the other isn't scoped right.

Since `layout` is already part of public API, we can't rename that. I'd recommend renaming the type to be `VideoGalleryLayout`

Reply:
okay this makes sense! will update that!